raft/server.cc

```diff
@@ -1441,8 +1452,8 @@ future<> server_impl::wait_for_apply(index_t idx, abort_source* as) {
         // This will be signalled when read_idx is applied
         auto it = _awaited_indexes.emplace(idx, awaited_index{{}, {}});
         if (as) {
-            it->second.abort = as->subscribe([this, it] () noexcept {
-                it->second.promise.set_exception(request_aborted());
+            it->second.abort = as->subscribe([this, it]() noexcept {
```

Comment:
nit: another whitespace removal

Reply:
I think these space removals come from the current clang-format auto-formatting (unfortunately I couldn't make the clang-format keep the space in the lambda when I was trying it in the past - seems it doesn't have options for it)

In this case you could remove the `()` completely though I think.

Reply:
Done, removed!